### ISSUE TITLE
ISSN 2167-776X | Journal of pastoral care & counseling (SAGE) | ORCID

### ISSUE THREAD

**Issue by IxKo** [ubtue/DatenProbleme#1620]
**URL**
IxTheo#2021-11-13#8688A2E8EEA100F3816EA27871FD90608447AF02
https://doi.org/10.1177/15423050211028189

**Ausführliche Problembeschreibung**
Die ORCID-ID (unterhalb des Rezensionstextes) ist nicht automatisch erfasst worden.

**Comment by IxKo** [comment]
Auf Nu mit Try url wird keine ORCID übertragen.

Im zotk bricht sowohl beim Multi- als auch Single-Import der Translator ubtue_SAGE Journals.js ab

**Comment by socheres** [comment]
Lieber @jriedl liebe @helenanebel, würdet Ihr Euch dieses Issue genauer ansehen? Im Scaffold wird ORCID übertragen, aber nicht auf Nu. Im Moment kann ich wegen Retrokat nicht auf Benu testen.

**Comment by socheres** [comment]
Liebe @IxKo, bitte in zotkat testen.

**Comment by IxKo** [comment]
Im zotkat-Verfahren mit Single-Import über ubtue_SAGE Journals kommt die ORCID nun mit.

**Comment by helenanebel** [comment]
Liebe @IxKo , 
es geht darum, die Anpassungen in den Zota-Translator zu übernehmen, richtig?
Viele Grüße, Helena Nebel

**Comment by IxKo** [comment]
@helenanebel 
Ja, im halbautomatischen Verfahren wird die ORCID nun importiert.
Auf Nu und ub28 ist das aber noch nicht der Fall.

**Comment by helenanebel** [comment]
Liebe @IxKo , lieber @jriedl , lieber @socheres ,
es handelt sich wohl um dasselbe Problem, das im Zusammenhang mit SAGE auch schon des öfteren im Meeting zur Sprache kam:
**HTTP request to https://journals.sagepub.com/doi/10.1177/15423050211028189 rejected with status 503
Feb 18 15:20:44 nu zts[1278]:   InternalServerError: An error occurred retrieving the document**
Das Problem liegt hier ganz klar nicht am Translator, da die Website schon nicht heruntergeladen werden kann.
Deshalb unassigne ich mich, es wäre nett, wenn sich @jriedl das mal ansehen könnte.
Viele Grüße, Helena Nebel

**Comment by IxKo** [comment]
Auf Nu und ub28 wird die ORCID-Angabe weiterhin nicht übernommen.

**Comment by jriedl** [comment]
Die letzten Änderungen von @socheres waren nur im zotkat-Repository. Für eine Übernahme nach zotaut müsste ein PR für ubtue/zotero-translators erstellt werden.

**Comment by mara-spiess** [comment]
Die Änderungen scheinen auch für ZotKat übernommen worden zu sein, @IxKo könnte das erneut getestet werden?

**Comment by IxKo** [comment]
Ich kann leider nicht testen, da beim Einzel-Import nicht der ubtue-Translator genommen wird, sondern aufgrund eines Fehlers der Standard-SAGE-Translator.
![image](https://github.com/user-attachments/assets/4ac8c6d1-c08c-4c4a-97ed-2ef8558c603f)

Im Mehrfachimport bricht der Vorgang ab.

Das scheint immer von bestimmten Einträgen hervorzugehen. Bei https://doi.org/10.1177/15423050211021388 ist bspw. dasselbe Probleme, aber andere Einträge funktionieren.

**Comment by mara-spiess** [comment]
Verwendet wird der Translator ubtue_SAGE Journals, richtig? Bei mir kann ich mit diesem den Fehler leider nicht rekreieren.

**Comment by mara-spiess** [comment]
@IxKo Könnte bitte überprüft werden ob der Fehler noch vorliegt?

**Comment by IxKo** [comment]
Einzel-Import ging wieder. Die ORCID wird importiert.
Allerdings wird sie in diesem Fall doppelt importiert und entsprechend exportiert: https://doi.org/10.1177/15423050211021388
![image](https://github.com/user-attachments/assets/382e467d-4977-41b6-8ad7-e1f7d2a32602)

**Comment by mara-spiess** [comment]
> Einzel-Import ging wieder. Die ORCID wird importiert. Allerdings wird sie in diesem Fall doppelt importiert und entsprechend exportiert: https://doi.org/10.1177/15423050211021388 ![image](https://github.com/user-attachments/assets/382e467d-4977-41b6-8ad7-e1f7d2a32602)

In Zusammenhang mit diesem Problem und dem Issue https://github.com/ubtue/DatenProbleme/issues/2256 wurde die Selektion von Orcids im Footer der Artikel wieder entfernt, da es bei dem in diesem Issue ursprünglich genannten Fall um eine Ausnahme gehandelt zu haben scheint und zu einer Reihe von Poblemen geführt hat.
Sollte sich dieser Fall doch wiederholen/häufen kann dieses Issue wieder geöffnet werden.
